copado-function/app/Retrieve.config

```diff
@@ -5,7 +5,7 @@
 * [Description] (Retrieve metadata from Marketing Cloud Repository) 
```

Comment:
Please add `installMcdevLocally` to this file

Reply:
Done